src/helpers_test.js

```diff
@@ -53,6 +53,16 @@ describe("getCourseNumbers", () => {
   })
 })
 
+describe("getCourseFeatureObject", () => {
+  it("returns the expected object from a course feature object", () => {
+    const featureObject = helpers.getCourseFeatureObject(
+      singleCourseJsonData["course_features"][2]
+    )
+    assert.equal(featureObject["feature"], "Assignments")
+    assert.equal(featureObject["subfeature"], "design with examples")
+  })
+})
```

Comment:
since there are null checks in the function, should there be a test for the case when those keys aren't there?

Reply:
added a test for this just now